tests/net/socket/socketpair/src/_main.c

```diff
@@ -4,15 +4,42 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include "_main.h"
+
 #include <zephyr/logging/log.h>
+#include <zephyr/net/socket.h>
 #include <zephyr/ztest.h>
 
-LOG_MODULE_REGISTER(net_test, CONFIG_NET_SOCKETS_LOG_LEVEL);
```

Comment:
Why are we changing the module names?

Reply:
`net_test` was completely non-descriptive of the testsuite. It was likely a typo.

Reply:
There was originally an idea to use net_test as a log module  name for network unit tests, dunno if it still applies. Note that the module name here is only used for logging purposes.

Reply:
You're right, that's it. I forgot, net subsys logging is actually very well templated.

Reply:
@jukkar - fixed!